docs/extensibility/extend-rest-api-formatters.md

```diff
@@ -0,0 +1,95 @@
+# Formatters
+As part of ExtendRestAPI, we introduced `Formatters`, these are utility classes that allow you to format values to
+so that they are compatible with the StoreAPI, values such as money, currency, or HTML.
+
+## How to use them
+To get a formatter, you can use the `get_formatter` method of the `ExtendRestApi` class. This method accepts a string,
+which is the name of the formatter you want to use, e.g. (money, html, currency).
+
+This returns a `FormatterInterface` which has the `format` method.
+
+The `format` method signature is:
+```php
+format( $value, array $options = [] );
+``` 
+Only `MoneyFormatter`'s behaviour can be controlled by the `$options` parameter. 
+
+## MoneyFormatter
+The [`MoneyFormatter`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/src/StoreApi/Formatters/MoneyFormatter.php)
+class can be used to format a monetary value using the store settings. The store settings may be overriden by passing
+options to this formatter's `format` method.
+
+### Arguments
+- `$value` - `number` - The number you want to format into a monetary vaue
+- `$options` - `Array`
+  - `decimals` - `integer` - Used to control how many decimal places should be displayed in the monetary value. Defaults
+  to the store setting.
+  - `rounding_mode` - `integer` - Used to determine how to round the monetary value. This should be one of the PHP
+  rounding modes described in the
+  [PHP round() documentation](https://www.php.net/manual/en/function.round.php). Defaults to `PHP_ROUND_HALF_UP`
+  
+### Example use and returned value
+```php
+get_formatter( 'money' )->format( 10.443, [
+  'rounding_mode' => PHP_ROUND_HALF_DOWN,
+  'decimals'      => 2
+] );
+```
+returns `1044`
```

Comment:
returns `10.44`

Reply:
During my testing, when using that example it does return `1044` not `10.44`.

If you add my example above the return in https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/0315fd59a7bba68818635fc5832dce39c7bea11b/src/StoreApi/Schemas/AbstractSchema.php#L259

and load the cart block you should see `1044`.